modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -161,120 +169,94 @@ function editIssue()
     emailUser($issueID, $issueValues['assignee']);
 
     return array(
-        'isValidSubmission' => true,
-        'issueID' => $issueID,
-    );
+            'isValidSubmission' => true,
+            'issueID'           => $issueID,
+           );
 }
 
-//also make this better
 /**
- * Validates those fields that need it
- * Currently PSCID and visitLabel as fk to session
+ * Validate PSCID and Visit Label
  *
- * @param array $validateValues values to be validated
- * @param int $issueID issue ID
+ * @param array $values  values to be validated
+ * @param int   $issueID issue ID
  *
  * @throws DatabaseException
  *
  * @return array success with values, or error with message.
  */
-function validateInput($validateValues, $issueID)
+function validateInput($values)
 {
-    $db =& Database::singleton();
-    $user =& User::singleton();
+    $db     =& Database::singleton();
+    $result = [
+               'PSCID'             => (isset($values['PSCID']) ? $values['PSCID'] : null),
+               'visit'             => (isset($values['visitLabel']) ? $values['visitLabel'] : null),
+               'candID'            => null,
+               'sessionID'         => null,
+               'isValidSubmission' => true,
+               'invalidMessage'    => null,
+              ];
+
+    // If both are set, return SessionID and CandID
+    if (isset($result['PSCID']) && isset($result['visit'])) {
```

Comment:
I think this could be made a bit more explicit/readable by using `elseif`:
i.e.
```
if (isset($result['PSCID']) && isset($result['visit'])) {
...
} elseif (isset($result['PSCID'])) {
...
} elseif (isset($result['visit'])) {
...
} else {
...
}
```